cinema_service/settings.py

```diff
@@ -20,12 +26,10 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
-    "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
-)
+SECRET_KEY = os.environ.get("SECRET_KEY", "default_secret_key")
```

Comment:
Ensure that the `SECRET_KEY` environment variable is set in your deployment environment to avoid using the default secret key.